ServiceBusWithClaimBasedAuthorization.csproj

```diff
@@ -9,28 +9,28 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.6' ">$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
-    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.6' ">1.6.1</NetStandardImplicitPackageVersion>
-    <TargetFrameworks>net452;netstandard1.6</TargetFrameworks>
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard2.0' ">$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
     <ApplicationIcon />
     <OutputTypeEx>exe</OutputTypeEx>
+    <LangVersion>8.0</LangVersion>
     <StartupObject />
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.19" />
+    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.0.0-preview.6" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.7.0" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.22" />
     <PackageReference Include="Microsoft.Azure.Management.Fluent" Version="1.30.0" />
```

Comment:
Generally speaking, I believe that we're looking to not highlight the Fluent library.   We should probably consider converting this.

Reply:
The use of Fluent library should be the core operation of this sample. Should we convert this? i think the code in RunSample() is this sample's goal.